inspectors/models/abstractinspectormodel.cpp

```diff
@@ -461,6 +461,6 @@ Ms::Score* AbstractInspectorModel::parentScore() const
     IF_ASSERT_FAILED(Ms::MuseScoreCore::mscoreCore) {
         return nullptr;
     }
-
-    return Ms::MuseScoreCore::mscoreCore->currentScore();
+    Ms::Score* returnScore = Ms::MuseScoreCore::mscoreCore->currentScore2() ? Ms::MuseScoreCore::mscoreCore->currentScore2() : Ms::MuseScoreCore::mscoreCore->currentScore();
```

Comment:
This function is not used anymore; in fact, it has been removed. It is not necessary to modify this file. Undoing this change will take care of the merge conflict.

Reply:
And since this is the only place `currentScore2()` is called, it seems we no longer have a need for the `cs2` member at all.

Reply:
Thank you very much (both for this comment and your PR). `cs2` is indeed unused right now, but I might have a use for it in the future.